code/modules/mob/new_player/new_player.dm

```diff
@@ -275,6 +275,9 @@
 		return 0
 	if(!job.player_old_enough(src.client))
 		return 0
+	if(job.species_whitelist.len)
```

Comment:
Is this check needed? The `Find()` won't return anything if it doesn't succeed anyways.